tutorials/CMakeLists.txt

```diff
@@ -871,10 +871,12 @@ if(ROOT_pyroot_FOUND)
       analysis/dataframe/df032_RDFFromNumpy.py
       math/fit/combinedFit.py
       math/fit/multifit.py
+      math/fit/NumericalMinimization.py
```

Comment:
Unrelated to this PR per se.  Do you know why we use `FILE(GLOB ....` here instead of just a hard coded list?

Reply:
I guess the strategy is copied from a different location, and the advantage of `GLOB` is obviously the support for expressions like:
`math/fit/*_numpy.py`

Reply:
Here is an example:
https://github.com/root-project/root/blob/d72ed64dfbbc38f4a5acec7003ac995a68e583d4/tutorials/CMakeLists.txt#L751

I guess people just use globs for all the vetos, so you can switch between list and globs.

Reply:
Glob are ... annoying in term of their state in regard to adding and removing files ... so we were moving away from them but this example looks new .. :(